lumber-update.js

```diff
@@ -0,0 +1,100 @@
+const program = require('commander');
+const chalk = require('chalk');
+const dotenv = require('dotenv');
+const inquirer = require('inquirer');
+const P = require('bluebird');
+const spinners = require('./services/spinners');
+const Database = require('./services/database');
+const DatabaseAnalyzer = require('./services/analyzer/database-analyzer');
+const Dumper = require('./services/dumper');
+const logger = require('./services/logger');
+const DiffDetector = require('./services/diff-detector');
+const { terminate } = require('./utils/terminator');
+const { ERROR_UNEXPECTED } = require('./utils/messages');
+
+program
+  .description('Check the difference between you actual configuration and your database and generate what is missing')
+  .parse(process.argv);
+
+async function askForConfirmation(spinner, message) {
+  spinner.pause();
+  const result = await inquirer.prompt([{
+    type: 'confirm',
+    name: 'confirm',
+    message,
+    default: true,
+  }]);
+  spinner.continue();
+  return result.confirm;
+}
+
+(async () => {
+  dotenv.config();
+  const config = {
+    appName: '',
+    dbConnectionUrl: process.env.DATABASE_URL,
+    dbSchema: process.env.DATABASE_SCHEMA,
+    dbSSL: process.env.DATABASE_SSL && JSON.parse(process.env.DATABASE_SSL),
+  };
+
+  let schema = {};
+
+  const connectionPromise = new Database().connect(config);
+  spinners.add('database-connection', { text: 'Connecting to your database' }, connectionPromise);
+  const connection = await connectionPromise;
+
+  const schemaPromise = new DatabaseAnalyzer(connection, config, true).perform();
+  spinners.add('database-analysis', { text: 'Analyzing the database' }, schemaPromise);
+  schema = await schemaPromise;
+
+  const diffSpinner = spinners.add('diff-check', { text: 'Check the difference with your actual models' });
+  const diffDetector = new DiffDetector({ sourceDirectory: '.' });
+  const newTables = diffDetector.detectNewTables(schema);
+  const newFields = diffDetector.detectNewFields(schema);
+  const newReferences = diffDetector.detectNewRelationships(schema);
```

Comment:
Shouldn't we also remove destroyed objects?
It is more critical IMO as the admin api server may crash if some columns/tables have been removed.
(It won't if some columns/tables have been added)

Reply:
Yes we should but it was much easier to add new fileds/models. We could do it in another PR don't you think ?